doc/classes/ConfigFile.xml

```diff
@@ -119,6 +158,22 @@
 				Deletes the specified key in a section. Raises an error if either the section or the key do not exist.
 			</description>
 		</method>
+		<method name="erase_section_key_comment">
+			<return type="void" />
+			<param index="0" name="section" type="String" />
+			<param index="1" name="key" type="String" />
+			<description>
+				Blanks out the comment pertaining to a specified key in a section. Raises an error if either the section or the key do not exist.
+			</description>
+		</method>
+		<method name="get_section_key_comment" qualifiers="const">
+			<return type="String" />
+			<param index="0" name="section" type="String" />
+			<param index="1" name="key" type="String" />
+			<description>
+				Returns the comment associated with the specified section and key. If either the section or the key do not exist, an error is raised, and an empty string is returned.
```

Comment:
Although they're admittedly a bit outdated and there's always exceptions, see the [writing guidelines](https://docs.godotengine.org/en/stable/contributing/documentation/docs_writing_guidelines.html).
"An error is raised, and an empty string is returned" should not use the passive voice, rather it should be active.

Godot's functions do not have exceptions that can be caught and handled, so "an error is raised" does not work here.
```suggestion
				Returns the comment associated with the specified section and key. If either the section or the key do not exist, this method fails and returns and an empty string.
```